src/main/kotlin/de/amplimind/codingchallenge/controller/AdminController.kt

```diff
@@ -51,6 +45,16 @@ class AdminController(
         return ResponseEntity.ok(userInfo)
     }
 
+    @Operation(summary = "Endpoint for deleting a user by email")
+    @ApiResponse(responseCode = "204", description = "User was deleted successfully.")
```

Comment:
Wieso kein 200? sicher dass es ein 204 is?

Reply:
Ich kanns auch auf 200 ändern, dachte aber 204 No Content würd besser passen

Reply:
Das Ding ist dieses @OpenApi... zeug ist nur Docu dass ändert an dem was Spring Boot zurück gibt nix^^
Die Respone was man bekommt hängt einzig davon ab was ResponeEntity für eine HttpStatus bekommt: und wenn du wie hier ResponseEntity.ok().build() machst du indirekt eig. `ResponseEntity.status(HttpStatus.OK)` was dir dann einen 200 Response liefert